frontend/src/pages/auth/auth.tsx

```diff
@@ -1,25 +1,29 @@
 import { AppRoute } from '~/libs/enums/enums.js';
 import {
   useAppDispatch,
-  useAppSelector,
   useCallback,
   useLocation,
 } from '~/libs/hooks/hooks.js';
-import { type UserSignUpRequestDto } from '~/packages/users/users.js';
+import {
+  type UserSignInRequestDto,
+  type UserSignUpRequestDto,
+} from '~/packages/users/users.js';
 import { actions as authActions } from '~/slices/auth/auth.js';
 
 import { SignInForm, SignUpForm } from './components/components.js';
+import styles from './styles.module.css';
 
 const Auth: React.FC = () => {
   const dispatch = useAppDispatch();
-  const { dataStatus } = useAppSelector(({ auth }) => ({
-    dataStatus: auth.dataStatus,
-  }));
+
   const { pathname } = useLocation();
 
-  const handleSignInSubmit = useCallback((): void => {
-    // handle sign in
-  }, []);
+  const handleSignInSubmit = useCallback(
```

Comment:
Do you really need useCallback here?🤔

Reply:
useCallback isn't really needed there, but it helps with the linter error:
35:28  error  JSX props should not use functions  react/jsx-no-bind
It doesn't allow any type of function(arrow, func. declaration, expression).